src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/pbs/PBSJobManager.groovy

```diff
@@ -73,21 +62,9 @@ class PBSJobManager extends ClusterJobManager<PBSCommand> {
     @Override
     boolean getDefaultForHoldJobsEnabled() { return true }
 
-    private List<String> collectJobIDsFromJobs(List<BEJob> jobs) {
-        BEJob.jobsWithUniqueValidJobId(jobs).collect { it.runResult.getJobID().shortID }
-    }
-
     @Override
-    void startHeldJobs(List<BEJob> heldJobs) {
-        if (!isHoldJobsEnabled()) return
-        if (!heldJobs) return
-        String qrls = "qrls ${collectJobIDsFromJobs(heldJobs).join(" ")}"
-
-        ExecutionResult er = executionService.execute(qrls)
-        if(!er.successful){
-            logger.warning("Hold jobs couldn't be started. \n status code: ${er.exitCode} \n result: ${er.resultLines}")
-            throw new Exception("Hold jobs couldn't be started. \n status code: ${er.exitCode} \n result: ${er.resultLines}")
-        }
+    protected ExecutionResult executeStartHeldJobs(List<BEJobID> jobIDs) {
+        executionService.execute("qrls ${jobIDs*.id.join(" ")}")
```

Comment:
I am not happy with this! In the original code, there was at least some checks and an Exception. Now "nothing" happens! Why did you remove it?

Reply:
Same applies for a lot of other commands.

Reply:
Checks and exception had been only moved to BEJobManager, because it's needed for all job managers